### DISCUSSION TITLE
ValueError: cannot reshape array of size 19200 into shape (244,324)

### DISCUSSION THREAD

**Discussion by gabrielpoli30** [bitcraze/discussions#865] [Q&A] [answered]
Hello,
I'm trying to run the AI-deck Face Detection example and I encountered the following error when running the command _python3 opencv-viewer.py_: crazyflie-clients-python/aideck-gap8-examples/examples/other/wifi-img-streamer/opencv-viewer.py", line 112, in <module> bayer_img.shape = (244, 324)
ValueError: cannot reshape array of size 19200 into shape (244,324)

![Captura de tela de 2023-08-10 15-54-23](https://github.com/bitcraze/discussions/assets/127146287/542880a8-a3bf-4ee0-9d35-a4a870cffd8f)

When I ran the same command on the WiFi Video Streamer, I didn't have any issues. I'm using AI-deck 1.1 with the Himax HM01B0 grayscale camera.

Can someone assist me with this error?

**Comment by knmcguire** [comment, marked as answer]
Hi! I think the facedetection has its own opencv-viewer since the size of the streamed image is different https://github.com/bitcraze/aideck-gap8-examples/blob/master/examples/image_processing/FaceDetection/opencv-viewer.py. As you noticed already 244*344 is not 19200 :)

**Reply by gabrielpoli30** [reply]
It worked, thank you very much! 
Does this example work with a color camera?

**Reply by knmcguire** [reply]
it kind of works... but technically shouldn't because of the Bayer filter. You will need to demosaick the image first and convert it to grayscale.

**Reply by gabrielpoli30** [reply]
Ah, I understand, thanks!
